core/src/Exceptions/CoreErrorMessages.cs

```diff
@@ -83,7 +83,7 @@ internal class CoreErrorMessages
         public const string ParsingWsTrustResponseFailed = "Parsing WS-Trust response failed";
         public const string UnknownUserType = "Unknown User Type";
 
-
-
+        public const string InternalErrorCacheEmptyUsername = "Internal error - trying to remove an ADAL user with an empty username. Possible cache corruption. See https://aka.ms/adal_token_cache_serialization";
```

Comment:
Merge concflicts ? Are these related to IWA / UP ?

Reply:
yes

---
In reply to: [219456012](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1261#discussion_r219456012) [](ancestors = 219456012)